src/PrestaShopBundle/Resources/views/Admin/Sell/Customer/view.html.twig

```diff
@@ -48,9 +48,7 @@
     </div>
   </div>
 
-  <div class="row">
-    {{ renderhook('displayAdminCustomers', {'id_customer': customerInformation.customerId.value}) }}
-  </div>
+  {{ renderhook('displayAdminCustomers', {'id_customer': customerInformation.customerId.value}) }}
```

Comment:
For this one I have doubts since it's only populated via a hook will it no break the display if modules expected to add a list of columns?

Reply:
@jolelievre You are right, if modules use boostrap col, they will have 15px extra padding on both sides. It should be considered as a small BC, but for the good purpose. I think the modules should not care about what is around the hook they hook into.

If we keep the .row and I just want to display a paragraph with my module, it will move it by -15px to both sides.

Reply:
Ok for me, then you just need to rebase (there is a conflict) And maybe remove the extra lines and it's ok for me!